tests/test_admin/test_utilities.py

```diff
@@ -87,8 +87,9 @@ def test_parse_csv_check_columns_mismatch(self):
                 expected_columns=", ".join(expected_columns), actual_columns=", ".join(header)
             )
 
-            with raises(ValidationError, message=expected_error_message):
+            with raises(ValidationError):
                 list(parse_csv(stream, expected_columns={"Name", "Email"}))
+                fail(expected_error_message)
```

Comment:
This is a case where the code wasn't doing what was intended; I think we want to switch the keyword argument on these from `message` to `match` instead of adding the `fail()` calls.  See https://docs.pytest.org/en/latest/deprecations.html#message-parameter-of-pytest-raises and https://docs.pytest.org/en/latest/reference.html#pytest-raises for more details.